web/src/js/__tests__/components/ContentView/ContentLoaderSpec.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import ContentLoaderGenerator from '../../../components/ContentView/ContentLoader'
```

Comment:
As far as I consider, the anonymous class in `ContentLoader` is just a wrapper, which takes a `View` component and attach some methods to it, it is not the actual content loader. So I named it `ContentLoaderGenerator` and its return component `ContentLoader`. I know this is wired, so I wonder if you have any suggestions?